unumpy/tests/test_numpy.py

```diff
@@ -264,6 +284,11 @@ def test_functions_coerce_with_dtype(backend, method, args, kwargs):
         (np.nonzero, ([3, 1, 2, 4],), {}),
         (np.where, ([[3, 1, 2, 4]],), {}),
         (np.gradient, ([[0, 1, 2], [3, 4, 5], [6, 7, 8]],), {}),
+        (np.split, ([1, 2, 3, 4], 2), {}),
+        (np.array_split, ([1, 2, 3, 4, 5, 6, 7, 8], 3), {}),
+        (np.dsplit, ([[[1, 2], [3, 4]], [[5, 6], [7, 8]]], 2), {}),
+        (np.hsplit, ([[1, 2], [3, 4]], 2), {}),
+        (np.vsplit, ([[1, 2], [3, 4]], 2), {}),
```

Comment:
These need to be xfailed for CuPy which doesn't accept 0 dim arrays as the section count.